x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/page.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState, useEffect } from 'react';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useUrlState } from '../../../util/url_state';
+import { NodeAvailableWarning } from '../../../components/node_available_warning';
+import { SavedObjectsWarning } from '../../../components/saved_objects_warning';
+import { UpgradeWarning } from '../../../components/upgrade';
+import { JobMap } from '../job_map';
+import { HelpMenu } from '../../../components/help_menu';
+import { useMlKibana, useMlApiContext } from '../../../contexts/kibana';
+import { useRefreshAnalyticsList } from '../../common';
+import { MlPageHeader } from '../../../components/page_header';
+import { AnalyticsIdSelector, AnalyticsSelectorIds } from '../components/analytics_selector';
+import { AnalyticsEmptyPrompt } from '../analytics_management/components/empty_prompt';
+
+export const Page: FC = () => {
+  const [globalState] = useUrlState('_g');
+  const [isLoading, setIsLoading] = useState(false);
+  const [jobsExist, setJobsExist] = useState(true);
+  const { refresh } = useRefreshAnalyticsList({ isLoading: setIsLoading });
+  const mapJobId = globalState?.ml?.jobId;
+  const mapModelId = globalState?.ml?.modelId;
+  const [analyticsId, setAnalyticsId] = useState<AnalyticsSelectorIds | undefined>();
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const {
+    dataFrameAnalytics: { getDataFrameAnalytics },
+  } = useMlApiContext();
+  const helpLink = docLinks.links.ml.dataFrameAnalytics;
+
+  const checkJobsExist = async () => {
+    try {
+      const { count } = await getDataFrameAnalytics(undefined, undefined, 0);
+      setJobsExist(count > 0);
+    } catch (e) {
+      // Swallow the error and just show the empty table in the analytics id selector
+      console.error('Error checking analytics jobs exist', e); // eslint-disable-line
+    }
+  };
+
+  useEffect(function checkJobs() {
+    checkJobsExist();
+  }, []);
+
+  const getEmptyState = () => {
+    if (jobsExist === false) {
```

Comment:
When you delete a DFA job, the model is not deleted. So you can show a map (probably with just one node) for a model for which the job has been deleted (you can open it from the Trained Models page for example). So maybe this should also check if any models created by DFA jobs exist. Or if that is not possible, don't use the empty prompt for the Map view?